awscli/examples/cloudwatch/put-insight-rule.rst

```diff
@@ -0,0 +1,34 @@
+**To create a contributor insights rule**
+
+The following ``put-insight-rule`` example creates a Contributor Insights rule named ``VPCFlowLogsContributorInsights`` in the specified account. If the command succeeds, no output is returned. ::
```

Comment:
Same here. The line for "This command produced no output." should be moved to after the command example where the output section is.
https://aws.github.io/aws-cli/docs_styleguide.html#command-produces-no-output